src/jobs/helpers/jobVerifier.js

```diff
@@ -33,6 +35,13 @@ module.exports.verifyJobBody = (req, res, next) => {
             errorToThrow.statusCode = 400;
         }
     }
+    const jobPlatform = await configHandler.getConfigValue(consts.CONFIG.JOB_PLATFORM);
+    if (jobPlatform === consts.AWS_FARGATE) {
+        if (!jobBody.tag) {
```

Comment:
need to add validation that tag exists in the custom definition configured in settings